onnxruntime/core/mlas/lib/platform.cpp

```diff
@@ -34,33 +34,27 @@ MLAS_PLATFORM MlasPlatform;
 #define _XCR_XFEATURE_ENABLED_MASK 0
 #endif
 
-inline
-uint64_t
```

Comment:
Can you undo the clang formatting here? MLAS is not using that style. I think you just should have the cut/paste of the GetSystemInfo from an earlier version of this file.